pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -6432,13 +7953,15 @@ def solve_model(model, options=None):
     if options["scale_model"] is True:
         # Scale model
         scaled_model = scale_model(model, scaling_factor=options["scaling_factor"])
-
-        # solve mathematical model
-        print("\n")
-        print("*" * 50)
-        print(" " * 15, "Solving scaled model")
-        print("*" * 50)
-        results = opt.solve(scaled_model, tee=True)
+        if model.config.water_quality is WaterQuality.discrete:
```

Comment:
Looking at this again, I think we need to add a call to solve_discrete_water_quality(model, opt) in the case the options["scale_model"] is False and model.config.water_quality is WaterQuality.discrete. Without this, when the scaling_model is false, the model solves without the strategic initialization of the solve_discrete_water_quality. 

Do correct me if I'm wrong!

I think this means that the else: (in line 7958) should have an if statement for `if model.config.water_quality is WaterQuality.discrete:` and `results = solve_discrete_water_quality(model, opt)`

https://github.com/project-pareto/project-pareto/blob/51b7fcdd7d0de2be7760a3cf4710163dbbedbeb5/pareto/strategic_water_management/strategic_produced_water_optimization.py#L7953-L7971